server/data/orchestrationApiTypes.ts

```diff
@@ -7,12 +7,14 @@ export type VisitorSupport = components['schemas']['VisitorSupportDto']
 // Temp fixes for enums incorrectly defined as strings in orchestration API - VB-2081
 // export type PageVisitDto = components['schemas']['PageVisitDto']
 // export type Visit = components['schemas']['VisitDto']
+// export type VisitHistoryDetails = components['schemas']['VisitHistoryDetailsDto']
 export type Visit = Omit<components['schemas']['VisitDto'], 'visitRestriction' | 'visitStatus' | 'visitType'> & {
   visitRestriction: 'OPEN' | 'CLOSED' | 'UNKNOWN'
   visitStatus: 'RESERVED' | 'CHANGING' | 'BOOKED' | 'CANCELLED'
   visitType: 'SOCIAL'
 }
 export type PageVisitDto = Omit<components['schemas']['PageVisitDto'], 'content'> & { content: Visit[] }
+export type VisitHistoryDetails = Omit<components['schemas']['VisitHistoryDetailsDto'], 'visit'> & { visit: Visit }
```

Comment:
All this can be tidied once https://dsdmoj.atlassian.net/browse/VB-2081 is resolved.